ds-prep-python-loops/main.py

```diff
@@ -0,0 +1,41 @@
+record = ("1", "Grimdiana", "Bones", "boulders")
+row = " "
```

Comment:
This isn't an empty string, as it has a space (`" "`) in it:

---

![image](https://user-images.githubusercontent.com/16143771/165217320-ed9e0969-9f53-4a2c-a0c0-2c25409af2da.png)